grails-app/controllers/au/org/ala/ecodata/CommentController.groovy

```diff
@@ -0,0 +1,161 @@
+package au.org.ala.ecodata
+
+import grails.converters.JSON
+import groovy.json.JsonSlurper
+
+class CommentController {
+    def outputService
+
+    def ignores = ["action","controller"]
+
+    def list() {
+        List comments = []
```

Comment:
You can use the Elvis operator as a shortcut for these sort of checks. e.g. ```params.sort ?: "dateCreated"```

Same effect, just a bit shorter and cleaner.

The ternary operation on sortOrder is unnecessary: just use ```boolean sortOrder = order == 'asc'```

Also, use types where known. In this case, it'll also catch invalid args straight up: e.g. if params.start is a string, ```int offset = params.start ?: 0``` will spit out an exception, but ```def offset =....``` won't.